authors/settings.py

```diff
@@ -134,10 +138,15 @@
 AUTH_USER_MODEL = 'authentication.User'
 
 REST_FRAMEWORK = {
-    'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
-    'NON_FIELD_ERRORS_KEY': 'error',
+    # 'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
+    # 'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
-    ),
+        'rest_framework.authentication.BasicAuthentication',
+        'rest_framework.authentication.SessionAuthentication',
+    )
+
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
```

Comment:
@kevpy Please revert this change and keep the JWT authentication setup. i.e remove the basic and session authentication values.